packages/cli-repl/test/e2e.spec.ts

```diff
@@ -203,6 +203,44 @@ describe('e2e', function() {
     }
   });
 
+  describe('set appName', () => {
+    context('with default appName', () => {
+      let shell;
+      beforeEach(async() => {
+        shell = TestShell.start({ args: [`mongodb://${await testServer.hostport()}/`] });
+        await shell.waitForPrompt();
+        shell.assertNoErrors();
+      });
+      it('appName set correctly', async function() {
+        if (process.env.MONGOSH_TEST_FORCE_API_STRICT) {
+          return this.skip(); // $currentOp is unversioned
+        }
+        const currentOp = await shell.executeLine('db.currentOp()');
+        expect(currentOp).to.include("appName: 'mongosh 0.0.0-dev.0'");
```

Comment:
@alenakhineika This is also breaking release CI right now -- just commenting so we don't forget about it :)